docs/content/app-protect-waf/configuration.md

```diff
@@ -19,7 +19,7 @@ The NGINX Ingress Controller has a set of global configuration parameters that a
 
 You can enable and configure NGINX App Protect WAF on the Custom Resources (VirtualServer, VirtualServerRoute) or on the Ingress-resource basis.
 
-To configure NGINX App Protect WAF on a VirtualServer resource, you would create a Policy Custom Resource referencing the APPolicy Custom Resource, and add this to the VirtualServer definition. See the documentation on the [App Protect WAF Policy](/nginx-ingress-controller/configuration/policy-resource/#waf).
+To configure NGINX App Protect WAF on a VirtualServer resource, you would create a Policy Custom Resource referencing the APPolicy Custom Resource or APPolicy Bundle, and add this to the VirtualServer definition. See the documentation on the [App Protect WAF Policy](/nginx-ingress-controller/configuration/policy-resource/#waf).
```

Comment:
```suggestion
To configure NGINX App Protect WAF on a VirtualServer resource, create a Policy Custom Resource referencing the APPolicy Custom Resource or APPolicy Bundle, and add this to the VirtualServer definition. Additional information can be found in the [police resource documentation](/nginx-ingress-controller/configuration/policy-resource/#waf).
```
`APPolicy` reads really awkwardly to me; I want to change it to something else but I can't grok enough context to figure out if this is an "official" noun or not.